engine/lib/metastrings.php

```diff
@@ -284,6 +279,14 @@ function _elgg_get_metastring_based_objects($options) {
 		$query = "SELECT {$options['metastring_calculation']}(v.string) as calculation FROM {$db_prefix}$type n_table";
 	}
 
+	foreach ($joins as $i => $join) {
+		if ($join === false) {
```

Comment:
I don't know the purpose of it, it's always been there.  I know I've used it as an undocumented feature to invalidate a query without hitting the database.